scripts/test-system-setup-loopback-only.py

```diff
@@ -39,7 +39,8 @@ async def test_connect(cmd: List[str]) -> bool:
     try:
         await asyncio.wait_for(proc.wait(), 10)
     except asyncio.TimeoutError:
-        return False
+        proc.terminate()
+        await proc.wait()
```

Comment:
I think you could keep the `return False` to make the failure explicit, but sigterm should make the return code 15 right?

Reply:
Yes, `-15`. I'll leave the `return False`.